src/twisted/newsfragments/9980.misc

```diff
@@ -0,0 +1 @@
+fix the readthedocs build, enable readthedocs PR builds and move readthedocs config into .readthedocs.yml
```

Comment:
I think that the release note should not read like a commit message, but rather as a brief / once sentence / non exhaustive summary / description of the changes in this PR.

So maybe just `Fix Read The Docs build and enabled PR builds.`

Reply:
```suggestion
Fix Read The Docs build and enabled PR builds.
```